packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.js

```diff
@@ -106,4 +118,10 @@ export default class QuanticSearchBoxSuggestionsList extends LightningElement {
   preventDefault(event) {
     event.preventDefault();
   }
+
+  get listboxCssClass() {
+    return `slds-dropdown slds-dropdown_length-5 slds-dropdown_fluid ${
+      this.suggestions?.length ? '' : ' slds-hidden'
```

Comment:
```suggestion
      this.suggestions?.length ? '' : 'slds-hidden'
```
You already have a space